### PR TITLE
Remove the description of the docker shim from the kubelet

### PR DESCRIPTION
**Pull request by yangjunmyfm192085**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Remove the description of the docker shim from the kubelet
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
"NONE"
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by odinuge — APPROVED** [review]
/triage accepted
/priority backlog
/lgtm

File: cmd/kubelet/app/server.go [hunk 1]
```diff
@@ -252,7 +252,7 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 			if err := checkPermissions(); err != nil {
 				klog.ErrorS(err, "kubelet running with insufficient permissions")
 			}
-			// set up signal context here in order to be reused by kubelet and docker shim
+			// set up signal context here in order to be reused by kubelet
 			ctx := genericapiserver.SetupSignalContext()
```
Comment (endocrimes) [inline cmd/kubelet/app/server.go:256]: now that we don't reuse the context we can actually move this to line 266, to keep it more closely scoped to the place it's used.

Comment (yangjunmyfm192085) [reply cmd/kubelet/app/server.go]: Hi, @endocrimes, thanks for review, I have applied all suggestions. Could you take a look again? thanks.

**Review by yangjunmyfm192085 — COMMENTED** [review]
/cc @endocrimes

**Review by endocrimes — APPROVED** [review]
/lgtm

**Pull request merged**